src/Chapter3.hs

```diff
@@ -344,6 +344,20 @@ of a book, but you are not limited only by the book properties we described.
 Create your own book type of your dreams!
 -}
 
+data ISBN = ISBN String
```

Comment:
Very useful!

Now as we learned newtypes, you can see that this is a perfect opportunity for one 😉